libs/anglify/src/modules/bottom-navigation/components/bottom-navigation-item/bottom-navigation-item.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="bottom-navigation-item" [class.active]="active$ | async">
+  <div class="icon" *ngIf="slots | findSlot: 'icon'">
+    <ng-template *anglifySlotOutlet="slots | findSlot: 'icon'"></ng-template>
+  </div>
+  <div class="label" *ngIf="(slots | findSlot: 'label') && (shift$ | async) && (active$ | async)">
+    <ng-template *anglifySlotOutlet="slots | findSlot: 'label'"></ng-template>
+  </div>
+  <div class="label" *ngIf="(slots | findSlot: 'label') && (shift$ | async) === false">
+    <ng-template *anglifySlotOutlet="slots | findSlot: 'label'"></ng-template>
+  </div>
+</div>
```

Comment:
Please also remove the div wrapper here and style the host directly instead. HostBindings will also be necessary